r1cs/gadgets/crypto/src/crh/pedersen/mod.rs

```
@@ -54,8 +54,14 @@ where
                 padded_input.push(UInt8::constant(0u8));
             }
         }
-        assert_eq!(padded_input.len() * 8, W::WINDOW_SIZE * W::NUM_WINDOWS);
-        assert_eq!(parameters.params.generators.len(), W::NUM_WINDOWS);
+        // TODO: check error message
```

Let's return the same error message we returned for the primitive